StanfordHCI/foundry
# Task status in the sidebar.

This is how the status in the side bar works:
1. You are assigned to Mockups V1. ( This shows up when the previous task is not finished yet.)
2. You can start Mockups V1. (when all previous tasks are done and the current task is ready to start)
3. Your task is in progress.
4. Your task is delayed.
5. You have finished all your tasks! (When all tasks assigned the worker are finished.)

The status text has the same color code as the events. 

How to test:
Create a team with two different workers. Try to have long task names. Assign one worker to each task and check if the status is shown correctly. Refresh the page and check the status again.
